nf_core/pipeline-template/tests/pipeline/default/main.nf.test

```diff
@@ -1,11 +1,12 @@
 nextflow_pipeline {
 
-    name "default"
+    name "Test Workflow main.nf"
     script "main.nf"
 
     test("default test config") {
         when {
             params {
+                outdir = "tests/results"
                 outdir = "$outputDir"
```

Comment:
Can you have two of the same param?

Reply:
so how about `outputDir = "tests/results"` ? (will have to test it though!)

Reply:
`outdir = "$outputDir"` is the only way in my opinion. Otherwise, tests become non-deterministic and can have side effects of being run one after another.

It's also easy to dig into individual tests executions as well.